infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
+STAGE = os.environ.get("STAGE")
+GH_HOOK_SECRET = os.environ.get("GH_HOOK_SECRET")
+
+sns = boto3.client("sns")
+
+
+class SNSTopicNotFoundError(Exception):
+    """Raised when trying to publish to a non-existent SNS topic"""
+    pass
+
+
+# e.g. "arn:aws:sns:eu-west-1:1234:dev_gh_team_events"
+def _topic_arn(gh_event):
+    return "{}:{}_gh_{}_events".format(
+        SNS_ARN,
+        STAGE,
+        gh_event,
```

Comment:
I like that, I didn't know you could do it.

Regarding the GH acronym is quite common and short and I thought in this context (user = developer) it was appropriate to use it. We can change everything to use the full `github` name if you think it's better.